Day5/PrintNumbers.java

```diff
@@ -0,0 +1,33 @@
+package Day5;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class PrintNumbers {
+    public static void main(String[] args) {
+        ExecutorService Threads = Executors.newFixedThreadPool(2);
+        
+        Threads.execute(()-> {
+            even();
+        });
+
+        Threads.execute(()-> {
+            odd();
+        });
+        
+        Threads.shutdown();
+    }
+
+    synchronized public static void even() {
```

Comment:
What happens when we remove the synchronized keyword?

Reply:
synchronised is a keyword used to create a monitor around the methods which allows only one thread at a time hence if it is removed both the threads will race to finish the task and end up with mixed outputs. Output without synchronised:

Printing Even numbers from 1 to 100
Printing Odd numbers from 1 to 100
 1 2 3 4 6 8 10 5 7 9 12 14 16 11 13 15 17 19 21 23 25 27 29 31 33 35 37 39 18 41 20 43 22 24 26 28 30 45 47 49 51 53 32 34 36 38 40 42 44 46 48 50 52 54 56 58 60 62 64 66 68 70 72 74 76 78 80 82 84 86 88 90 55 92 94 96 98 100
 57 59 61 63 65 67 69 71 73 75 77 79 81 83 85 87 89 91 93 95 97 99